apps/fhir/server/loggers.py

```diff
@@ -1,38 +1,29 @@
-import json
-import logging
-
-from django.conf import settings
+import apps.logging.request_logger as logging
 
 
 """
   Logger functions for fhir/server module
 """
-match_fhir_id_logger = logging.getLogger('audit.authenticate.match_fhir_id')
 
 
-# For use in authentication.log_match_fhir_id()
-def log_match_fhir_id(auth_flow_dict, fhir_id, mbi_hash, hicn_hash,
+def log_match_fhir_id(request, fhir_id, mbi_hash, hicn_hash,
                       match_found, hash_lookup_type, hash_lookup_mesg):
     '''
         Logging for "fhir.server.authentication.match_fhir_id" type
         used in match_fhir_id()
     '''
-    log_dict = {
+    match_fhir_id_logger = logging.getLogger(logging.AUDIT_AUTHN_MATCH_FHIR_ID_LOGGER, request)
+    match_fhir_id_logger.info({
         "type": "fhir.server.authentication.match_fhir_id",
-        "auth_uuid": auth_flow_dict.get('auth_uuid', None),
-        "auth_app_id": auth_flow_dict.get('auth_app_id', None),
-        "auth_app_name": auth_flow_dict.get('auth_app_name', None),
-        "auth_client_id": auth_flow_dict.get('auth_client_id', None),
-        "auth_pkce_method": auth_flow_dict.get('auth_pkce_method', None),
         "fhir_id": fhir_id,
         "mbi_hash": mbi_hash,
         "hicn_hash": hicn_hash,
         "match_found": match_found,
         "hash_lookup_type": hash_lookup_type,
         "hash_lookup_mesg": hash_lookup_mesg,
-    }
-
-    if settings.LOG_JSON_FORMAT_PRETTY:
-        match_fhir_id_logger.info(json.dumps(log_dict, indent=2))
-    else:
-        match_fhir_id_logger.info(json.dumps(log_dict))
+        # pre populate with auth_xxx to satisfy unittests
+        "auth_app_id": None,
```

Comment:
Is this something we should be adding into the `RequestLogger` class? I feel like the defaults should be consistent across all logging.

Reply:
can be added in request_logger extract_request_data(...)

Reply:
Yeah I just think we should make it consistent. So if it doesn't exist should it be `None`? Otherwise if what we are doing now for those is the norm, we should update the tests to expect that format instead. I actually think @dtisza1 mentioned something about what is better for logging in splunk.